.config-schema.json

```diff
@@ -21,7 +21,7 @@
         "maxConcurrentReadsForCheck": {
             "description": "The maximum allowed number of concurrent reads in a single Check query (default is MaxUint32).",
             "type": "integer",
-            "default": 4294967295,
+            "default": 100,
```

Comment:
?

Reply:
As noted in https://github.com/openfga/openfga/pull/1963#discussion_r1788122391, we were not using this config value and config is about to be removed.  I repurpose the variable for the purpose of defining how many concurrent child branch for userset.  Since we use this variable for channel length, concurrent routine etc., this variable needs to be a sane value rather than -1.

Reply:
Sorry I wasn't clear - the config value **is** being used, but not in LocalChecker, it's being used in `CheckQuery`. You will need to pass it down to LocalChecker.

Reply:
Fixed so that we use concurrencyLimit instead.  As such, the variable value does not need to change now.